cmu_graphics/__init__.py

```diff
@@ -1,7 +1,13 @@
 import __main__
 import os
 
-main_directory = os.path.dirname(os.path.abspath(__main__.__file__))
+try:
+    main_directory = os.path.dirname(__main__.__file__)
```

Comment:
it looks like you removed the `os.path.abspath` from this line. If there wasn't a particular reason you removed it, I think we should keep it.

Reply:
Well, it just seemed to give back the same thing with and without `os.path.abspath`, but sure lets keep it.